components/engine_rocks/src/ttl_properties.rs

```diff
@@ -74,6 +74,7 @@ pub struct TtlPropertiesCollector<F: KvFormat> {
 
 impl<F: KvFormat> TablePropertiesCollector for TtlPropertiesCollector<F> {
     fn add(&mut self, key: &[u8], value: &[u8], entry_type: DBEntryType, _: u64, _: u64) {
+        // DBEntryType::BlobIndex will be skit because we can't parse the value.
```

Comment:
```suggestion
        // DBEntryType::BlobIndex will be skipped because we can't parse the value.
```
How about add a check in config validation that ttl can't enabled with titan at the same time.